### PR TITLE
Appcenter integration

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
Appcenter integration

## Links
[MINI-2010](https://jira.rakuten-it.com/jira/browse/MINI-2010)

# Checklist
- [X] I have read the [contributing guidelines](CONTRIBUTING.md).
- [X] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: MiniApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -558,13 +559,32 @@
...
+			inputPaths = (
+			);
+			name = "Set RMAHostAppUserAgentInfo";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "bundleShortVersionString=$(/usr/libexec/PlistBuddy -c \"print:CFBundleShortVersionString\" \"$CONFIGURATION_BUILD_DIR/$INFOPLIST_PATH\")\nversionString=$(/usr/libexec/PlistBuddy -c \"print:CFBundleVersion\" \"$CONFIGURATION_BUILD_DIR/$INFOPLIST_PATH\")\n/usr/libexec/PlistBuddy -c \"Set RMAHostAppUserAgentInfo MiniApp Demo App/$bundleShortVersionString\" \"$CONFIGURATION_BUILD_DIR/$INFOPLIST_PATH\"\n";
```
Comment (Climbatize) [inline MiniApp.xcodeproj/project.pbxproj:587]: I just splitted a scripts that was doing 2 different actions

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: Example/Info.plist [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>AppCenterSecret</key>
+	<string>$(RMA_APP_CENTER_SECRET)</string>
```
Comment (corycaywood) [inline Example/Info.plist:6]: Did you also add this secret to the `prod` build job on App Center?

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: Example/Info.plist (hunk 2 above)
Comment (Climbatize) [reply Example/Info.plist:6]: yes it is :)

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**